api/api_sources/schema-migration-sql/CountryProvinceSchema/CountryProvinceSchema.sql

```diff
@@ -0,0 +1,37 @@
+-- ### Creating Table: country_province ### --
+
+        
+CREATE TABLE country_province ();
+ALTER TABLE country_province ADD COLUMN province_code VARCHAR(2) NOT NULL;
+ALTER TABLE country_province ADD COLUMN country_code VARCHAR(3) NOT NULL;
+ALTER TABLE country_province ADD COLUMN description VARCHAR(100) NULL;
+
+
+        
+-- ### Creating Comments on table ### --
+
+        
+COMMENT ON TABLE country_province IS 'Standard ISO-3166 code table for listed countries and provinces';
```

Comment:
It might make sense to update the table comment to include provinces, states, territories, etc.

Suggestion: _"Standard ISO-3166 code table for listed countries and sub-divisions such as provinces, states, territories, etc."_